indispensable-asn1/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/ObjectIdentifier.kt

```diff
@@ -159,7 +159,8 @@ class ObjectIdentifier @Throws(Asn1Exception::class) private constructor(
         }
 
         /**
-         * Casts out the evil demons that haunt OID components encoded into [rawValue]
+         * Casts out the evil demons that haunt OID components encoded into ASN.1 content [rawValue].
+         * If you want to parse human-readable OID representations, just use the ObjectIdentifier constructor!
```

Comment:
rename to `parseASN1` instead of having smart kdoc

Reply:
now aligned with how corresponding methods in integer, etc. are named